src/ray/protobuf/gcs_service.proto

```diff
@@ -177,6 +177,13 @@ service ActorInfoGcsService {
   rpc KillActorViaGcs(KillActorViaGcsRequest) returns (KillActorViaGcsReply);
 }
 
+message GetClusterIdRequest {}
+
+message GetClusterIdReply {
+  GcsStatus status = 1;
```

Comment:
Do we need this? isn't it always OK?

Reply:
I think having Status is convention, you can see other Replys in here that even have a "not populated" comment. Always useful for future evolution when Status will be necessary.